UpdateNode/index.ts

```diff
@@ -0,0 +1,96 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import { Document } from 'mongoose';
+import authWrapper from '../shared/authWrapper';
+import { ForbiddenFunctionError, ObjectNotFoundFunctionError, ConflictError, httpErrorHandler, ValidationFunctionError } from '../shared/error/error';
+import HttpTriggerResult from '../shared/httpTriggerResult';
+import { ValidationError } from 'joi';
+import { Role, User } from '../shared/user/user.interface';
+import { Node, NodeType } from '../shared/node/node.interface';
+import NodeModel from '../shared/node/node.model';
+import NodeGroup from '../shared/nodeGroup/nodeGroup.interface';
+import NodeGroupModel from '../shared/nodeGroup/nodeGroup.model';
+import { Event } from '../shared/event/event.interface';
+import EventModel from '../shared/event/event.model';
+import getConnection from '../shared/mongoose';
+import updateUnitSchema from './joi';
+
+const functionName = 'updateNode';
+
+const checkFutureEventsOfNode = async (nodeId: String) => {
+    const futureEvents: Event[] = await EventModel.find({ 'node.id': nodeId, time: { $gte: new Date() } }).exec();
+    if (futureEvents) {
+        const withInterviewers = futureEvents.filter((eventObject) => eventObject.interviewersIds.length);
+        if (withInterviewers.length) {
+            throw new ConflictError(functionName, 'There are future events with interviewers');
+        }
+    }
```

Comment:
```suggestion
   if (futureEvents?.some((eventObject) => eventObject.interviewersIds.length)) {
       throw new ConflictError(functionName, 'There are future events with interviewers');
   }
```

Why? In that case, we improve our best-case scenario & one future event with interviewers should be enough for us. we don't need to find all of those invalid events. 
I think that there is a way to search if `interviewersIds` is empty in the same request.

Reply:
I don't agree. You assume that future events existence mean that the interviewers ID is full, but you don't know that... maybe the interviewers are added just one day before?